webpack.config.js

```diff
@@ -105,6 +107,28 @@ module.exports = {
 				},
 			} ) )
 		),
+
+		new CopyWebpackPlugin(
+			[
+				'block-editor',
+				'editor',
+				'components',
+				'edit-post',
```

Comment:
How do you decide which package to include here?
Do we want this plugin to be applied to the CSS files shipped in the npm packages or is it meant to only affect the CSS files that ship in the plugin and Core?

Reply:
> How do you decide which package to include here?

Again, just sort of grabbed the files that I know new needed it here. I'm thinking everything except `block-library/` should be super strong in specificity. 
One that I missed that should be added here is the `format-library/` styles. Also, maybe `edit-widgets/`?

> Do we want this plugin to be applied to the CSS files shipped in the npm packages or is it meant to only affect the CSS files that ship in the plugin and Core?

Another great question and one I'll need some guidance on. My initial thought is that the added specificity would be helpful in any use-case.